integreat_cms/core/middleware/region_middleware.py

```diff
@@ -69,23 +72,16 @@ def get_available_regions(request, user_regions):
         :type request: ~django.http.HttpRequest
```

Comment:
Shouldn't the description of this method just be `This method returns the regions available to the user`? The request is only used to get the user object. The request itself has no influence on the result, beyond authentication?!

Reply:
Ah no, I overlooked the line `else request.available_regions`